packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -113,7 +113,7 @@ const convertToUiBlockedMessage = (
   return {
     messageType: 'blocked',
     createdOn: message.createdOn,
-    warningText: message.content?.message,
+    warningText: undefined,
```

Comment:
Why is this undefined? We are not taking from message anymore?

Reply:
By default the ChatSDK will send empty string based on https://learn.microsoft.com/en-us/azure/communication-services/how-tos/chat-sdk/data-loss-prevention workaround, and this will cause it to not have a default message. So by setting it to empty string it will have default message